docs/guide-firebase-user.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Guide: Firebase implementation
+sidebar_label: Guide: Firebase implementation
+---
+
+Firebase is used for sample implementation of _**user login**_ and _**data storing**_.
+
+---
+
+**Note:** This part is completely _**optional**_ and can be replaced with any other solution of your choice.
+
+_**Firebase**_ and _**Google login**_ related `.env` variables can be left _**empty**_. In this case the app will operate normally, only the user login part will be _**disabled**_.
```

Comment:
So this is why you're checking for the length of firebase.apps?

Reply:
Exactly. As mentioned in other comment, I've linked this description in `isFirebaseAvailable` notes to make it clearer.